test-suite/src/metadata/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod table;
+pub use table::table;
```

Comment:
These can provide two different ways to use module.

Like other modules we are using,
ref. https://github.com/gluesql/gluesql/blob/main/test-suite/src/data_type/mod.rs

How about only using `pub mod table;`?